front/app/components/IdeaCards/IdeasWithFiltersSidebar/ButtonWithFiltersModal/FiltersModal/BottomBar.tsx

```diff
@@ -16,37 +16,35 @@ import messages from './messages';
 interface Props {
   onClick: (event: FormEvent) => void;
   selectedIdeaFilters: InputFiltersProps['selectedIdeaFilters'];
-  filtersActive: boolean;
   onReset: () => void;
 }
 
-const BottomBar = memo<Props>(
-  ({ onClick, selectedIdeaFilters, onReset, filtersActive }) => {
-    const { data: ideasFilterCounts } =
-      useIdeasFilterCounts(selectedIdeaFilters);
-
-    if (!ideasFilterCounts) return null;
-
-    return (
-      <Box
-        background={colors.white}
-        p="16px"
-        pb="0"
-        flex="1"
-        borderTop={`1px solid ${colors.grey300}`}
-      >
-        <Button onClick={onClick} fullWidth={true}>
-          <FormattedMessage
-            {...messages.showXResults}
-            values={{
-              ideasCount: ideasFilterCounts.data.attributes.total,
-            }}
-          />
-        </Button>
-        <ResetFiltersButton onClick={onReset} filtersActive={filtersActive} />
-      </Box>
-    );
-  }
-);
-
+const BottomBar = ({ onClick, selectedIdeaFilters, onReset }: Props) => {
```

Comment:
Also removed `memo`. The props we pass to `BottomBar` are not memoized where they're passed down, so they are considered new props each time `BottomBar` is rendered, (which means `memo` is not changing anything).